modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -1279,6 +1279,8 @@ CV__DNN_INLINE_NS_BEGIN
          CV_WRAP void detect(InputArray frame, CV_OUT std::vector<int>& classIds,
                              CV_OUT std::vector<float>& confidences, CV_OUT std::vector<Rect>& boxes,
                              float confThreshold = 0.5f, float nmsThreshold = 0.0f);
+        private:
+            void init();
```

Comment:
Can we avoid adding it to a public header?

Reply:
Do you have any ideas?

Reply:
Due this is just a procedure we can just define a static method in the `model.cpp` like
```cpp
void disableNMS(Net& net)
{
    for (String& name : net.getUnconnectedOutLayersNames())
    {
        int layerId = net.getLayerId(name);
        Ptr<RegionLayer> layer = net.getLayer(layerId).dynamicCast<RegionLayer>();
        if (!layer.empty())
        {
            layer->nmsThreshold = 0;
        }
    }
}
```